src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -49,6 +50,7 @@ export class CommentEditorComponent implements OnInit {
 
   dragActiveCounter = 0;
   uploadErrorMessage: string;
+  @Input() maxLength = ISSUE_BODY_SIZE_LIMIT;
```

Comment:
since we've defined the issue body limit as a constant and it's unlikely that we're going to pass in another limit, i think the `@Input` decorator here is unnecessary. in the future if we somehow decide that another comment field needs another input limit then we can add it back in.

Reply:
Thank you for the comment! I'm just wondering if there's any disadvantage to putting the `@Input` decorator compared to removing it?

Reply:
functionally there's no difference, but my concern is that other devs might see the decorator and get a bit confused as they're expecting it to be used as an input in another component. thanks!

Reply:
Okay, I have removed the decorator!

Reply:
thanks, and LGTM! a gentle reminder to limit your commit message to 70 characters per line, once that's done then i'll approve this PR